include/sharg/detail/format_tdl.hpp

```diff
@@ -286,13 +287,13 @@ class format_tdl : format_base
         }
         for (size_t i{1}; i < executable_name.size(); ++i)
         {
-            auto name = "subcommand_" + std::to_string(i);
-            info.params.push_back(tdl::Node{
-                .name = name,
-                .value = tdl::StringValue(executable_name[i]),
-            });
-            info.cliMapping.emplace_back("", name);
+            parameters = {tdl::Node{
+                .name = executable_name[executable_name.size() - i],
+                .tags = {"basecommand"},
```

Comment:
I have no real clue what's going on. But if the tag here is `basecommand` and a value is given, shouldn't at least one test have a value at the `basecommand` output below?

Reply:
`basecommand` is a special tag (we have other special tags, like `output` or `directory`). It marks that the name of this node/section has to be attached to the command line.